pkg/operator/ceph/csi/operator_driver_test.go

```diff
@@ -38,70 +38,126 @@ import (
 )
 
 func TestReconcileCSI_createOrUpdateDriverResources(t *testing.T) {
-	ns := "test"
-	r := &ReconcileCSI{
-		context: &clusterd.Context{
-			Clientset:     testop.New(t, 1),
-			RookClientset: rookclient.NewSimpleClientset(),
-		},
-		opManagerContext: context.TODO(),
-		opConfig: opcontroller.OperatorConfig{
-			OperatorNamespace: "test",
-		},
-	}
-	cluster := &cephv1.CephCluster{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "testCluster",
-			Namespace: ns,
-		},
-		Spec: cephv1.ClusterSpec{
-			CSI: cephv1.CSIDriverSpec{
-				ReadAffinity: cephv1.ReadAffinitySpec{
-					Enabled:             true,
-					CrushLocationLabels: []string{"kubernetes.io/hostname"},
-				},
-				CephFS: cephv1.CSICephFSSpec{
-					KernelMountOptions: "ms_mode=crc",
-				},
-			},
-			Network: cephv1.NetworkSpec{
+	tests := []struct {
+		name               string
+		networkSpec        cephv1.NetworkSpec
+		expectMultusAnnots bool
+	}{
+		{
+			name: "without multus",
+			networkSpec: cephv1.NetworkSpec{
 				Connections: &cephv1.ConnectionsSpec{
 					Encryption: &cephv1.EncryptionSpec{
 						Enabled: true,
 					},
 				},
 			},
+			expectMultusAnnots: false,
+		},
+		{
+			name: "with multus",
+			networkSpec: cephv1.NetworkSpec{
+				Provider: "multus",
+				Selectors: map[cephv1.CephNetworkType]string{
+					"public":  "rook-ceph/rook-ceph-public-network",
+					"cluster": "rook-ceph/rook-ceph-cluster-network",
```

Comment:
CSI components do not need (and should not get) the cluster network annotation. Only OSDs get this annotation.

Reply:
My bad. This is the input config, not output.